Assets/Mirror/Runtime/Transport/Websocket/Server.cs

```diff
@@ -131,7 +133,7 @@ async Task ProcessTcpClient(TcpClient tcpClient, CancellationToken token)
                 WebSocketHttpContext context = await webSocketServerFactory.ReadHttpHeaderFromStreamAsync(tcpClient, stream, token);
                 if (context.IsWebSocketRequest)
                 {
-                    WebSocketServerOptions options = new WebSocketServerOptions() { KeepAliveInterval = TimeSpan.FromSeconds(30), SubProtocol = "binary" };
+                    WebSocketServerOptions options = new WebSocketServerOptions() { KeepAliveInterval = KeepAliveTimer == 0 ? TimeSpan.Zero : TimeSpan.FromSeconds(KeepAliveTimer), SubProtocol = "binary" };
```

Comment:
```suggestion
                    WebSocketServerOptions options = new WebSocketServerOptions() { KeepAliveInterval = TimeSpan.FromSeconds(Mathf.Max(0, KeepAliveTimer)), SubProtocol = "binary" };
```

Reply:
This is done at the transport level, would you suggest to do it here or at transport I think having both would be redundant.